grammar.js

```diff
@@ -163,6 +163,12 @@ module.exports = grammar({
     keyword_preserve: _ => make_keyword("preserve"),
     keyword_unsigned: _ => make_keyword("unsigned"),
     keyword_zerofill: _ => make_keyword("zerofill"),
+    keyword_on_conflict: _ => make_keyword("on conflict"),
```

Comment:
we already have `keyword_on`. Would be better to use `seq($.keyword_on, ...)

Reply:
Done